Staccato-iOS/Staccato-iOS/Presentation/View/SignIn/SignInView.swift

```diff
@@ -8,8 +8,55 @@
 import SwiftUI
 
 struct SignInView: View {
+    
+    @State private var nickName: String = ""
+    
     var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
+        NavigationView {
+            VStack {
+                Image("staccato_login_logo")
+                    .imageScale(.large)
```

Comment:
좀 더 정확한 사이즈 조절을 위해 아래와 같이 해보는 것은 어떨까요?
```suggestion
                    .resizable()
                    .scaleToFit()
                    .frame(width: [피그마에 정의된 값])
                    혹은
                    .padding([좌우 값을 이용한 사이즈 조절])
```